src/witan/send/adroddiad/valid_states.clj

```diff
@@ -2,40 +2,42 @@
   (:require [tablecloth.api :as tc]
             [witan.send.domain.academic-years :as ay]))
 
-(defn keystage-min-ay [y]
+(defn phase-min-ay [y]
   (cond
-    (ay/early-years y) (apply min ay/early-years)
-    (ay/key-stage-1 y) (apply min ay/key-stage-1)
-    (ay/key-stage-2 y) (apply min ay/key-stage-2)
-    (ay/key-stage-3 y) (apply min ay/key-stage-3)
-    (ay/key-stage-4 y) (apply min ay/key-stage-4)
-    (ay/key-stage-5 y) (apply min ay/key-stage-5)
-    (ay/ncy-15+ y) (apply min ay/ncy-15+)))
+    (ay/nursery y) (apply min ay/nursery)
+    (ay/primary-school y) (apply min ay/primary-school)
+    (ay/secondary-school y) (apply min ay/secondary-school)
+    (ay/post-16 y) (apply min ay/post-16)
+    (ay/post-19 y) (apply min ay/post-19)))
 
-(defn keystage-max-ay [y]
+(defn phase-max-ay [y]
   (cond
-    (ay/early-years y) (apply max ay/early-years)
-    (ay/key-stage-1 y) (apply max ay/key-stage-1)
-    (ay/key-stage-2 y) (apply max ay/key-stage-2)
-    (ay/key-stage-3 y) (apply max ay/key-stage-3)
-    (ay/key-stage-4 y) (apply max ay/key-stage-4)
-    (ay/key-stage-5 y) (apply max ay/key-stage-5)
-    (ay/ncy-15+ y) (apply max ay/ncy-15+)))
+    (ay/nursery y) (apply max ay/nursery)
+    (ay/primary-school y) (apply max ay/primary-school)
+    (ay/secondary-school y) (apply max ay/secondary-school)
+    (ay/post-16 y) (apply max ay/post-16)
+    (ay/post-19 y) (apply max ay/post-19)))
```

Comment:
do we want to separate p16 and p19 here?

Reply:
Yes, I think so, as a special school might have a sixth form attached, so it will only go to Key Stage 5, whereas FEC, can technically take any post-16 age, so there is a distinction